_posts/2016-11-29-future.md

```diff
@@ -0,0 +1,308 @@
+
+### Documentation/Website
+
+Docs contributions have definitely been lacking in the [last year](https://github.com/babel/babel.github.io/graphs/contributors?from=2015-11-29&to=2016-11-29&type=c).
+
+However just recently we've done a lot of awesome stuff:
+
+- [@Daniel15](https://github.com/Daniel15) has been maintaining [babel-standalone](https://github.com/babel/babel-standalone) which we use in the REPL and has some automation when new releases get out.
+- [@maxiloc](https://github.com/maxiloc) added search functionality via Algolia via [#977](https://github.com/babel/babel.github.io/pull/977)
+
+and added some new collaborators: 
+
+- [@STRML](https://github.com/STRML): Added discourse to all github pages via [#875](https://github.com/babel/babel.github.io/pull/875)
+- [@xtuc](https://github.com/xtuc): Added support for reading the README from the babel repo so we don't have to sync 2 copies of docs via [#990](https://github.com/babel/babel.github.io/pull/990)
+- [@fredericmarx](https://github.com/fredericmarx): Added support for a copy to clipboard button for code snippets via [#998](https://github.com/babel/babel.github.io/pull/998)
+- [@seedofjoy](https://github.com/seedofjoy): Added a resize ability for the REPL via [#1003](https://github.com/babel/babel.github.io/pull/1003)
+
+Interesting Issues:
+
+- [Reorganizing the docs layout](https://github.com/babel/babel.github.io/issues/930)
+- [Showcase community plugins](https://github.com/babel/babel.github.io/issues/831)
+- [Adding a resources page](https://github.com/babel/babel.github.io/issues/27)
+- [Learn ES2015: teach others about ES2015+](https://github.com/babel/babel.github.io/issues/994)
+- [REPL: adding example dropdowns](https://github.com/babel/babel.github.io/issues/59)
+- [REPL: supporting all options](https://github.com/babel/babel.github.io/issues/92)
+- [REPL: allow the REPL to import any community plugin from npm](https://github.com/babel/babel.github.io/issues/858)
+- [REPL: Use ASTexplorer](https://github.com/fkling/astexplorer/issues/70)
+- [ASTexplorer: Real Time collaboration](https://github.com/fkling/astexplorer/issues/166)
+
+## Future
+
+### [Plugin API Changes](https://github.com/babel/notes/blob/master/2016-08/august-01.md#potential-api-changes-for-traversal)
+
+There is a lot of confusion around how plugins/presets interact regarding ordering. This results in bugs and issues with the config which require users to place plugins before/after others in a non-intuitive way.
+
+This is a fundamental change to the core of Babel which will take some work to figure out.
+
+### [Versioning]
+
+TODO
+
+### TypeScript Support
+
+Be able to understand TypeScript syntax (like we do with Flow) and add a plugin to strip TypeScript types for better interop. This would just mean parsing TypeScript specific syntax and stripping it out. Anything more would require a lot more work and discussion.
```

Comment:
Just stripping out types won't work because TS adds new syntax, for example `enum`